### ISSUE TITLE
Design menu option B--change get started language--for usability testing

### ISSUE THREAD

**Issue by AmyKort** [18F/fec-cms#1762]
What we're after: We've learned from users  that our menus for the data, legal resources and H4CC pages might not be as effective as we'd like in getting folks to the resources they need. We're going to try some A/B testing on alternatives to see if we can learn more about what works for users. 

Option B is to change the "get started" language in the menu. 
![image.png](https://images.zenhubusercontent.com/595a678361b4ca3b0a731109/492074cc-ca01-49b7-a44b-51e7e2d14226)

We're going to try changing the language to see if that helps. Completion criteria:

- [x] Draft new languge
- [x] Incorporate into design

**Comment by AmyKort** [comment]
Looks like we've landed on 'Learn more" for the language change.

cc @patphongs @JonellaCulmer @jenniferthibault

**Comment by patphongs** [comment]
During design sync, we mind melded on some ideas for testing. There's some subtle differences that we added in here to see how users will do as well as if they will notice the heading links to go to the landing pages more. We didn't change any of the menu options, just added some small text change and visual cues. Here's what we did:

**Change 1:**
Added `»` as call to action for the header link

**Change 2:**
Changed button to `Learn more` as suggested by the content team

**Change 3:**
Slight tweaks in the language for the submenu descriptor text because some of them said Learn more already. We chose words like "Explore" and "Find"

<img width="478" alt="screen shot 2018-02-09 at 4 15 31 pm" src="https://user-images.githubusercontent.com/12799132/36050540-dff33ef6-0db4-11e8-9a7c-5deb16f858e8.png">
<img width="527" alt="screen shot 2018-02-09 at 4 17 59 pm" src="https://user-images.githubusercontent.com/12799132/36050541-e0034d8c-0db4-11e8-83c7-ee90fd1d2742.png">
<img width="521" alt="screen shot 2018-02-09 at 4 18 05 pm" src="https://user-images.githubusercontent.com/12799132/36050542-e011987e-0db4-11e8-81a1-79f669596132.png">
<img width="430" alt="screen shot 2018-02-09 at 4 18 11 pm" src="https://user-images.githubusercontent.com/12799132/36050543-e020b7dc-0db4-11e8-9c34-a46e0064c3d4.png">
